docs_user/modules/proc_adopting-image-service-with-nfs-backend.adoc

```diff
@@ -119,7 +119,7 @@ EOF
 . Patch the `OpenStackControlPlane` CR to deploy the {image_service} with an NFS back end:
 +
 ----
-$ oc patch openstackcontrolplane openstack --type=merge --patch-file glance_nfs_patch.yaml
+$ oc patch openstackcontrolplane openstack-galera-network-isolation --type=merge --patch-file glance_nfs_patch.yaml
```

Comment:
This was called out in a few procedures. But `openstack` should not be changed to `openstack-galera-network-isolation` in every case, right?

Link to hackfest comment: https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABQ54c_1g

Reply:
I'll change it back to `openstack`. Do you want to follow up with Partheeban?